.github/workflows/generate-demo-preview.yml

```diff
@@ -9,6 +9,7 @@ on:
     paths:
       - '.github/workflows/generate-demo-preview.yml'
       - 'config/**/*'
+      - 'dev/**/*'
```

Comment:
This should be also done in other workflows (we miss it when we move the demo code out of the lib #1324), but as this is not needed here, we can do the change in another PR.